templates/stripes/stripe-attributes.php

```diff
@@ -0,0 +1,14 @@
+<div class="<?= $this->e($type, 'wrapperClasses') ?>">
+  <div class="vssl-stripe-column">
+    <div class="vssl-stripe--attributes-fieldwrap">
+      <?php foreach ($values as $key => $value) : ?>
+        <div class="vssl-stripe--attributes-field vssl-stripe--attributes--<?= $key ?>">
```

Comment:
Second class should probably be `vssl-stripe--attributes-field--<?= $key ?>`, yeh?

Reply:
Ugh, yeh. We've ended up with a strange variation on BEM-style classes that eschews underscores. Based on other templates, I'd recommend many more double dashes in this code.

For example:
```
<div class="vssl-stripe--attributes--fieldwrap">
  <div class="vssl-stripe--attributes--field vssl-stripe--attributes--<?= $key ?>-field">
    ...
```